app/src/routes/routesMap.js

```diff
@@ -264,6 +265,7 @@ const routesMap = {
     payload: { ...payload, settingsTab: GENERAL },
   })),
   [PROJECT_SETTINGS_TAB_PAGE]: `/:projectId/settings/:settingsTab/:subTab*`,
+  [PROJECT_SETTINGS_TAB_PAGE_WITH_SUBPAGE]: `/:projectId/settings/:settingsTab\\?subPage=:subPage*`,
```

Comment:
I suggest to avoid specifying query parameters in routes as they may vary and the list can be extended without touching the route. In this case you are not added the `id` parameter that is also supported by integrations page.
So please remove this new route and all related code.